### DISCUSSION TITLE
What intervals are supported by Quartz.NET?

### DISCUSSION THREAD

**Discussion by PatVax** [quartznet/quartznet#2200] [Q&A] [answered]
I am developing an application that needs to send messages via a Hardware interface every 20ms (LIN protocol). I tested with .NET built in timers but they can't achieve my 20ms. I started to develop my own timer using Thread.Yield and Thread.SpinWait and while doing some googling on the matter I stumbled upon Quartz.NET. So if there already is a solution for my problem I would rather not reinvent the wheel.
So my question is can Quartz.NET reliably deliver 20ms precision while running indefinitely? Possibly multiple hours. Also for future reference, how far can Quartz.NET be pushed? Can I go to microsecond precision? What are the limits? I searched the website for this information but didn't find anything. I would appreciate if someone with experience using Quartz.NET could elaborate on this topic.
Thanks in advance

**Comment by lahma** [comment, marked as answer]
Quartz.NET has too many abstraction for it be used in such scenario where you want constant millisecond accuracy. You are probably better of just creating your own infinite loop with the `SpinWait` approach.
